yfinance/data.py

```diff
@@ -71,6 +79,49 @@ def _get_proxy(self, proxy):
         return proxy
 
     def get_raw_json(self, url, user_agent_headers=None, params=None, proxy=None, timeout=30):
-        response = self.get(url, user_agent_headers=user_agent_headers, params=params, proxy=proxy, timeout=timeout)
-        response.raise_for_status()
+        retry = 0
+        response = None
+
+        while response is None or (retry < self.max_retries and 400 <= response.status_code):
+            response = self.get(url, user_agent_headers=user_agent_headers, params=params,
+                                proxy=proxy, timeout=timeout)
+            print(retry)
+            retry += 1
+
+        if 400 <= response.status_code:
+            utils.warn_for_status(response)
```

Comment:
Replaced `raise` with `warn` so it doesn't throw the entire bulk operation in the event that some of the calls failed